# Available commands to use the Fisher Scientific PPS4102 Top Pan balance 
int32 ON = 0
int32 OFF = 1
int32 RESET = 2
int32 REVERSE = 3
int32 DISPENSEINDEF = 4
int32 DISPENSEINDEFREV = 5
int32 DISPENSE = 6
int32 DISPENSEREV = 7

int32 dispenser_command
float32 dispenser_speed
int32 dispenser_time
